CHANGELOG.md

```diff
@@ -111,9 +146,17 @@ versions (e.g. `find_package(SEQAN3 3.0.1)` requires at least seqan3 with a vers
 
 ## Notable Bug-fixes
 
+* All our headers are self contained now (#1085).
+* The alignment algorithm with edit distance returns the correct back coordinate now (#1093)
+* Inserting and deleting gaps in an empty seqan3::gap_decorator doesn't assert anymore (#1109).
```

Comment:
```suggestion
* Inserting and deleting gaps in an empty seqan3::gap_decorator don't assert anymore (#1109).
```

Reply:
I change it to `Inserting or deleting gaps in an empty seqan3::gap_decorator does not cause assert anymore (#1109).` which I think is correct?